plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/packagejson/N4JSProjectSetupJsonValidatorExtension.xtend

```diff
@@ -370,7 +360,8 @@ public class N4JSProjectSetupJsonValidatorExtension extends AbstractJSONValidato
 	}
 	
 	/** IDEBUG-266 issue error warning on cyclic dependencies. */
-	@Check
+	// TODO: GH-1500: Check if cyclic dependencies are already found in BuildManager#sortByDependencies()
+	// @Check // Commented out because there are performance issues.
```

Comment:
Is it intentional to merge this TODO to master?

Reply:
This should be done when working on #1525. I'll change the issue id.